www-apps/redmine/redmine-3.3.4.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-# ruby23 don't have required deps
+# ruby24 don't have required deps
```

Comment:
This is irrelevant change in the first commit. Use `git rebase -i` to edit it and amend it back to ruby23. Then update it in third.